ingress/multi-cluster/mci-https-backend/haproxy/haproxy.cfg

```diff
@@ -0,0 +1,23 @@
+global
+  stats socket /var/run/api.sock user haproxy group haproxy mode 660 level admin expose-fd listeners
+  log stdout format raw local0 debug
+
+defaults
+  mode http
+  timeout client 660s
+  timeout connect 60s
+  timeout server 60s
+  timeout http-request 660s
+  timeout http-keep-alive 660s
+  log global
+  maxconn 20000
+
+frontend demo-frontend
+    bind *:80
+    bind *:443 ssl crt /usr/local/etc/haproxy/mycert.pem
```

Comment:
Where is the .pem stored ? in a ConfigMap or Secret ? It would be better if it's stored as a secret actually

Reply:
Changed to secret.